test/integration/client-side-encryption/client_side_encryption.prose.14.decryption_events.test.ts

```diff
@@ -75,11 +75,14 @@ describe('14. Decryption Events', metadata, function () {
       keyId: keyId,
       algorithm: 'AEAD_AES_256_CBC_HMAC_SHA_512-Deterministic'
     });
-    // Copy ``ciphertext`` into a variable named ``malformedCiphertext``.
-    // Change the last byte to 0. This will produce an invalid HMAC tag.
+    // Copy ``ciphertext`` into a variable named ``malformedCiphertext``. Change the
+    // last byte to a different value. This will produce an invalid HMAC tag.
     const buffer = Buffer.from(cipherText.buffer);
-    buffer.writeInt8(0, buffer.length - 1);
+    const lastByte = buffer.readInt8(buffer.length - 1);
```

Comment:
```suggestion
    const lastByte = buffer.readUInt8(buffer.length - 1);
```

Figured it makes sense to be consistent with the sign (or just use bracket access)